### DISCUSSION TITLE
[VSCode build error] cc1.exe : unrecognized command line option '-Wpedantic'

### DISCUSSION THREAD

**Discussion by JorisFranck** [Estia-1a/projetGenieInfo_public#80] [Project Installation and Configuration] [answered]
**Bug description**
Lors du build avec CMake , erreur lors du build 

- What are the error messages you get in the console ?

 [ 20%] Building C object CMakeFiles/freud.dir/src/argsparse.c.obj
[build] cc1.exe: error: unrecognized command line option '-Wpedantic'
[build] make.exe[3]: *** [CMakeFiles/freud.dir/src/argsparse.c.obj] Error 1
[build] make.exe[2]: *** [CMakeFiles/freud.dir/all] Error 2
[build] make.exe[1]: *** [CMakeFiles/freud.dir/rule] Error 2
[build] make.exe: *** [freud] Error 2
[build] Build finished with exit code 2

**Step to reproduce**
Appuyer sur build après avoir mis 
cmake_minimum_required(VERSION 3.15)

***Configuration***
gcc version 4.6.1

Résolu en supprimant  -Wpedantic dans 
target_compile_options(freud PRIVATE -Wall -Wextra -Wpedantic)

**Screen shot**
![Capture d’écran 2022-06-13 090455](https://user-images.githubusercontent.com/98878408/173298124-e589d9d8-ea21-4af1-883c-fc34367a4724.png)

**Comment by dhmmasson** [comment, marked as answer]
Nice description of the bug 🎊 , this is not due to cmake but to the version of the compiler you are using. 

# Steps to Solve 
Remove -Wpedantic from line 16
```cmake 
target_compile_options(freud PRIVATE -Wall -Wextra) 
```
